awx/main/tests/functional/api/test_inventory.py

```diff
@@ -412,11 +412,16 @@ def test_delete_inventory_host(delete, host, alice, role_field, expected_status_
 # See companion test in tests/functional/test_rbac_inventory.py::test_inventory_source_update
 @pytest.mark.parametrize("start_access,expected_status_code", [(True, 202), (False, 403)])
 @pytest.mark.django_db
-def test_inventory_update_access_called(post, inventory_source, alice, mock_access, start_access, expected_status_code):
+def test_inventory_update_access_called(post, discover_available_cloud_provider_plugin_names, alice, mock_access, start_access, expected_status_code):
     with mock_access(InventorySource) as mock_instance:
         mock_instance.can_start = mock.MagicMock(return_value=start_access)
-        post(reverse('api:inventory_source_update_view', kwargs={'pk': inventory_source.id}), {}, alice, expect=expected_status_code)
-        mock_instance.can_start.assert_called_once_with(inventory_source)
+        post(
+            reverse('api:inventory_source_update_view', kwargs={'pk': discover_available_cloud_provider_plugin_names.id}),
+            {},
+            alice,
+            expect=expected_status_code,
+        )
+        mock_instance.can_start.assert_called_once_with(discover_available_cloud_provider_plugin_names)
```

Comment:
This check expects arguments passed to the tested mock, not a callable.

Reply:
This has been resolved via this commit: 1173ccb7be32f9991729b39e1ffe819505ef6218

Reply:
ack